ImageChecks.ps1

```diff
@@ -0,0 +1,167 @@
+param (
+    [String]$Image,
+    [String]$ConfigFile
+)
+
+[String]$resourcesDir = "$pwd\UnattendResources"

```

Comment:
if you run this script from another directory, pwd will mean something else.
never use pwd, use this:
https://github.com/cloudbase/windows-openstack-imaging-tools/blob/master/WinImageBuilder.psm1#L5